smithy-swift-codegen/src/test/kotlin/AuthSchemeResolverGeneratorTests.kt

```diff
@@ -10,14 +10,14 @@ class AuthSchemeResolverGeneratorTests {
     @Test
     fun `test auth scheme resolver generation`() {
         val context = setupTests("auth-scheme-resolver-generator-test.smithy", "com.test#Example")
-        val contents = getFileContents(context.manifest, "/Example/AuthSchemeResolver.swift")
+        val contents = getFileContents(context.manifest, "Sources/Example/AuthSchemeResolver.swift")
```

Comment:
Many tests follow.  In general, file locations are updated, and code is updated with namespaced symbols.  No further comments on tests.